src/api/lib/tasks/workflows.rake

```diff
@@ -0,0 +1,33 @@
+namespace :workflows do
```

Comment:
The purpose of this rake task is to only clean the data, right? After this PR is merged do we still need this task or the workflow_runs will change their state as expected? :slightly_smiling_face:

Reply:
Once this PR is merged, we'll need to manually run this rake task in production.
After that, all the workflow_runs affected by the same failure will change their status.

The task can be useful in case new similar failures happen again in the future.